go.mod

```diff
@@ -1,29 +1,31 @@
 module github.com/couchbase/gocbcoreps
 
-go 1.19
+go 1.23.0
+
+toolchain go1.24.5
```

Comment:
Unsure this is compatible with the current go version in the gocb go.mod

Reply:
Have changed it to go 1.23.12 to match gocb's go.mod